Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.h

```diff
@@ -14,6 +14,8 @@
 
 @interface Bluepill : NSObject
 
+@property (nonatomic, readonly) NSString *simulatorUDID;
```

Comment:
BPConfiguration already has "@property (nonatomic, strong) NSString *useSimUDID;"

simulatorUDID is defined as
- (NSString *)simulatorUDID {
    return self.context.runner.UDID;
}

Which is the runtime UDID and could be different than the one specified from the CLI.

Currently simulatorUDID is mostly used in the test cases